pkg/controller/status/conditions.go

```diff
@@ -23,6 +23,8 @@ const (
 	RemoteConfigurationAvailable configv1.ClusterStatusConditionType = "RemoteConfigurationAvailable"
 	// RemoteConfigurationInvalid is a condition type providing info about remote configuration content validity
 	RemoteConfigurationValid configv1.ClusterStatusConditionType = "RemoteConfigurationValid"
+	// DisabledWithAPI is a condition providing information about the disabling of gathering with the API.
+	DisabledWithAPI configv1.ClusterStatusConditionType = "DisabledWithApi"
```

Comment:
I would call the condition probably as `DataGatherDisabled` or `DataGatheringDisabled` or `GatheringDisabled`. WDYT?

Reply:
`GatheringDisabled` sounds good to me